File: 백준/Bronze/11720. 숫자의 합/숫자의 합.cc
#include <stdio.h>

int main()
{
    int N = 0, result = 0;
    scanf("%d", &N);
    char arr[101];
    scanf("%s", arr);
    for(int i = 0; i < N; i++)
    {
        result += arr[i] - '0';
    }
    printf("%d", result);
    return 0;
    
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        String arr = scanner.next();
        int result = 0;
        
        for(int i = 0; i < N; i++)
        {
            result += arr.charAt(i) - '0';    
        }
        System.out.print(result);
        scanner.close();
    }
}